yt/data_objects/static_output.py

```diff
@@ -339,6 +342,18 @@ def force_periodicity(self, val=True):
             raise TypeError("force_periodicity expected a boolean.")
         self._force_periodicity = val
 
+    def set_periodicity(self, val):
```

Comment:
```suggestion
    def set_periodicity(self, val: Union[bool, Tuple[bool, bool, bool]]) -> None:
```

Reply:
I assume it could be useful to accept `None` too so we're allowed to reset the initial state using only public API

Reply:
Yep, agreed!